recipe_scrapers/hungryhappens.py

```diff
@@ -0,0 +1,29 @@
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import get_equipment
+
+
+class HungryHappens(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "hungryhappens.net"
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".wprm-recipe-ingredient-group h4",
+            ".wprm-recipe-ingredient",
+        )
+
+    def equipment(self):
+        equipment_container = self.soup.select_one(".wprm-recipe-equipment-container")
+        if not equipment_container:
+            return []
```

Comment:
Although I don't think all recipes mention every item of equipment typically required, an empty list could seem to imply that no kitchen equipment (or at least no special kitchen equipment) is required for a specific recipe; in this case I think it's better to return `None` instead.

```suggestion
            return None
```